services/housetables/src/main/java/com/linkedin/openhouse/housetables/dto/mapper/UserTableVersionMapper.java

```diff
@@ -15,11 +15,21 @@
  */
 @Mapper(componentModel = "spring")
 public class UserTableVersionMapper {
+  public static final String INITIAL_VERSION = "INITIAL_VERSION";
 
   @Named("toVersion")
   public Long toVersion(UserTable userTable, @Context Optional<UserTableRow> existingUserTableRow) {
     if (!existingUserTableRow.isPresent()) {
-      return 1L;
+      if (!userTable.getTableVersion().equals(INITIAL_VERSION)) {
+        throw new EntityConcurrentModificationException(
+            String.format(
+                "databaseId : %s, tableId : %s %s",
+                userTable.getDatabaseId(),
+                userTable.getTableId(),
+                "The requested user table has been deleted by other processes."),
+            new RuntimeException());
+      }
+      return null;
```

Comment:
Whats the long value that gets persisted for a new entry?

Reply:
0